tools/arm_pack_manager/index.json

```diff
@@ -480222,6 +480222,74 @@
             ]
         ],
         "sub_family": "STM32WL55"
+    },
+    "STM32WLE5JCIx": {
+        "name": "STM32WLE5JCIx:CM4",
+        "memories": {
+          "IRAM1": {
+            "access": {
+              "read": true,
+              "write": true,
+              "execute": false,
+              "peripheral": false,
+              "secure": false,
+              "non_secure": false,
+              "non_secure_callable": false
+            },
+            "start": 536870912,
+            "size": 65536,
+            "startup": false,
+            "default": true
+          },
+          "IROM1": {
+            "access": {
+              "read": true,
+              "write": false,
+              "execute": true,
+              "peripheral": false,
+              "secure": false,
+              "non_secure": false,
+              "non_secure_callable": false
+            },
+            "start": 134217728,
+            "size": 262144,
+            "startup": true,
+            "default": true
+          }
+        },
+        "algorithms": [
+          {
+            "file_name": "CMSIS/Flash/STM32WLxx_CM4.FLM",
+            "start": 134217728,
+            "size": 262144,
+            "default": true,
+            "ram_start": null,
+            "ram_size": null
+          }
+        ],
+        "processor": {
+          "Symmetric": {
+            "units": 1,
+            "core": "CortexM4",
+            "fpu": "None",
+            "mpu": "Present"
+          }
+        },
+        "from_pack": {
+          "vendor": "Keil",
+          "pack": "STM32WLxx_DFP",
+          "version": "1.0.0",
```

Comment:
1.1.0 ?

Reply:
correct, it's fixed, may be should be also done for WL55 (nucleo)?

Reply:
not mandatory, as MCU from NUCLEO was already available in 1.0.0, I think